### ISSUE TITLE
Learning Rate Schedules

### ISSUE THREAD

**Issue by bnels** [DeepLearnPhysics/lartpc_mlreco3d#15]
It will likely be useful to have a way to schedule learning rates
e.g. in `*.cfg`
```yaml
training:
   learning_rate: [0.01, 0.001, 0.0001]
   iterations: [1000, 1000, 1000]
```
would do 1k iterations with learning rate at 0.01, then 1k iterations at learning rate 0.001, and so on

**Comment by dkoh0207** [comment]
Would it be better to try having support for the Pytorch learning rate schedulers in the config files? The schedulers work by doing scheduler.step(), similarly with the optimizer. 

[Pytorch Schedulers](https://pytorch.org/docs/stable/optim.html)

I may try working on this soon but feel free to take over, especially since I'm not yet too familiar with the details of lartpc_mlreco3d.

**Comment by bnels** [comment]
In #47, the approach suggested by @dkoh0207 has been implemented
